### PR TITLE
add isort

### PR DESCRIPTION
**Pull request by ostcar**
See also #1386

When all is ready, I will call the command "python make isort" to sort all imports. This are the only manual changes.

### CODE REVIEW

**Comment by ostcar** [comment]
The Travis Fail was expected. It will work when you give your OK and I actually changed the import-order.

**Comment by ostcar** [comment]
See also: #1386

**Comment by ostcar** [comment]
See also: https://groups.google.com/forum/#!topic/django-developers/EQhbI4cDuX4

**Comment by ostcar** [comment]
I changed the pull request that only isort is installed and a isort-config file is appended. With this, we can use isort to fix the imports in one file but do not have to change any file right now.

We can add the travis check when the release is near.

-> Ready for review.

**Comment by ostcar** [comment]
@normanjaeckel please review (I don't know If you realised my change here)

**Comment by ostcar** [comment]
@normanjaeckel push

File: setup.cfg [hunk 1]
```diff
@@ -0,0 +1,4 @@
+[isort]
+multi_line_output=3
```
Comment (normanjaeckel) [inline setup.cfg]: line-width is missing. And, if you like, you can order the options alphabetically.

Comment (ostcar) [inline setup.cfg]: line-width=79 is the default and does not have to be set in this file

**Comment by normanjaeckel** [comment]
Rebase and merge.

**Pull request merged**